curp/src/server/raw_curp/mod.rs

```diff
@@ -328,6 +342,68 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
         (info, Ok(()))
     }
 
+    /// Handle `propose_conf_change` request
+    pub(super) fn handle_propose_conf_change(
+        &self,
+        conf_change: ConfChangeEntry,
+    ) -> ((Option<ServerId>, u64), Result<(), ConfChangeError>) {
+        debug!(
+            "{} gets conf change for with id {}",
+            self.id(),
+            conf_change.id()
+        );
+
+        let st_r = self.st.read();
```

Comment:
Shall we block all the other commands from fast path by putting conf_change to the spec_pool, and make it conflict with all the other commands?

Reply:
We can only insert `Command` into `spec_pool`, but `conf_change` is't a `Command`. and the `conf_change` vertex is conflict with other vertexes, their execution order will not be swapped.

Reply:
If conf_change can't be inserted into `spec_pool`, does it go through the CURP protocol? I think it should.

Reply:
Curp itself does not have the ability of membership change. This ability is provided by the back-end raft. It can be understood that whether or not `conf_change` conflicts with other requests, it goes directly into slow path and waits for the completion of the back-end raft protocol before returning to the client. So there is no need to insert `conf_change` into the `spec_pool`